app/controllers/authenticate_controller.rb

```diff
@@ -95,7 +96,8 @@ def authenticator_input
       username:           params[:id],
       credentials:        request.body.read,
       client_ip:          request.ip,
-      request:            request
+      request:            request,
+      encoded_token:      params[:encoded] == "true"
```

Comment:
Fine with it. What do you say @orenbm ?

Reply:
How about ..../authenticate/encoded

Reply:
i strongly agree with @uCatu that it should be `.../authenticate?encoded`.
regarding the second endpoint please further investigate

Reply:
Which brings us to previous comments here... What's the behaviour when '.../authenticate?encoded=false` or any value to encoded?

Reply:
i think it's ok to document it just with `?encoded` and assume that if it used incorrectly (e.g `?encoded=false`) then it will not behave as expected.

i think it's ok to not support `encoded=false`